### ISSUE TITLE
predicates are in the wrong field in the "merged" KG2pre JSON-lines file?

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX-KG2#417]
In the artifact `kg2-merged-2.10.1-edges.jsonl` produced in the KG2.10.1pre build, the `predicate` field seems to be `null`, as pointed out by Kaiwen He on team Unsecret Agent:
```
{
"domain_range_exclusion": false, 
"id": "CHEMBL.COMPOUND:CHEMBL425---biolink:biolink_in_clinical_trials_for---None---None---None---MONDO:0005306---ClinicalTrialsKG:", 
"negated": false, 
"object": "MONDO:0005306", 
"predicate": null, 
"primary_knowledge_source": "ClinicalTrialsKG:", 
"publications": [], "publications_info": {}, 
"qualified_object_aspect": null, 
"qualified_object_direction": null, 
"qualified_predicate": null, 
"relation_label": "biolink:in_clinical_trials_for", 
"source_predicate": "biolink:biolink_in_clinical_trials_for", 
"subject": "CHEMBL.COMPOUND:CHEMBL425", 
"update_date": "1996-05-01"}
```

**Comment by saramsey** [comment]
@sundareswarpullela says that there was no issue with the edges TSV file that was used to build KG2.10.1c. Need to figure out whether the edges TSV file is build from the JSONL file, and if so, why there was no issue with the TSV file even though there appears to be an issue with the JSONL file.

**Comment by saramsey** [comment]
The second problem is that for some reason, the correct Biolink predicate information is provided in the "relation_label" field in the `kg2-merged-2.10.1-edges.jsonl` file, which seems weird.

**Comment by saramsey** [comment]
But now I recall that the merged file is pre-assignment of the final biolink predicate, so this seems OK, that the "predicate" field in `kg2-merged-2.10.1-edges.jsonl` would be `null`. Thus, I'm going to close this issue.
